app/Http/Transformers/UserTransformer.php

```diff
@@ -23,6 +23,8 @@ public function transform(User $user)
             'mobile' => null,
             'signup' => null,
             'reportback' => null,
+            'created_at' => $user->created_at->toIso8601String(),
```

Comment:
Not sure, but we expose `created_at` and `updated_at` in Northstar so figured I'd follow suit.